src/Microsoft.DotNet.Cli.Utils/Reporter.cs

```diff
@@ -20,8 +20,8 @@ private Reporter(AnsiConsole console)
         }
 
         public static Reporter Output { get; } = Create(AnsiConsole.GetOutput);
-        public static Reporter Error { get; } = Create(AnsiConsole.GetError);
-        public static Reporter Verbose { get; } = CommandContext.IsVerbose() ? Create(AnsiConsole.GetError) : Null;
+        public static Reporter Error { get; } = Create(AnsiConsole.GetOutput);
```

Comment:
It's nicer to be collate output from the error stream to do pretty printing on TeamCity. Why change this?